main/bamboo/bamboo.go

```diff
@@ -184,11 +184,11 @@ func listenToMarathonEventStream(conf *configuration.Configuration, sub api.Even
 	client := &http.Client{}
 	client.Timeout = 0 * time.Second
 
-	for _, marathon := range conf.Marathon.Endpoints() {
-		ticker := time.NewTicker(1 * time.Second)
-		eventsURL := marathon + "/v2/events"
-		go func() {
-			for _ = range ticker.C {
+	ticker := time.NewTicker(1 * time.Second)
+	go func() {
```

Comment:
Since the endpoints may change, I needed to allow this loop to call `conf.Marathon.Endpoints()` again after all endpoints have had failures. The prior code also would connect to all valid supplied endpoints. It is my understanding that this is not necessary-- with my change only one endpoint is connected at a time.